src/pages/dho/Home.vue

```diff
@@ -116,106 +114,26 @@ export default {
       },
       skip () { return !this.selectedDao || !this.selectedDao.docId },
       variables () { return { daoId: this.selectedDao.docId, first: 4 } }
-    },
-
-    upvoteElection: {
-      query: require('~/query/upvote-election-data.gql'),
-      update: data => {
-        const election = data?.getDao
-        const previousElection = election?.previouselct[0]
-        const ongoingElection = election?.ongoingelct[0]
-        const upcomingElection = election?.upcomingelct[0]
-        const currentRound = ongoingElection?.currentround[0]
-
-        return {
-          isActive: previousElection?.length > 1 || ongoingElection?.length > 1 || upcomingElection?.length > 1,
-          currentRound: currentRound?.details_type_s,
-          nextRound: currentRound?.nextround,
-          startTime: upcomingElection?.details_startDate_t,
-          endTime: currentRound?.details_endDate_t,
-          upcomingElection
-        }
-      },
-      skip () { return !this.selectedDao || !this.selectedDao?.name },
-      variables () { return { daoName: this.selectedDao?.name } }
     }
   },
 
   data () {
     return {
-
-      counterdown: undefined,
-      endDate: '2023-03-20',
       pagination: {
         first: 6,
         offset: 0,
         fetchMore: true
       },
       textFilter: null,
-      order: ordersMap[0],
-      currentUpvoteStep: null,
-      UPVOTE_DOC_URL
+      order: ordersMap[0]
     }
   },
 
   computed: {
     ...mapGetters('dao', ['daoSettings', 'selectedDao', 'selectedDaoPlan']),
 
-    currentStepIndex () {
-      let stepIndex = null
-      if (this.upvoteElection?.upcomingElection?.length) {
-        stepIndex = 0
-      } else if (!this.upvoteElection?.nextRound?.length && this.upvoteElection?.currentRound !== 'head') {
-        stepIndex = 4
-      } else {
-        switch (this.upvoteElection?.currentRound) {
-          case ('delegate'):
-            stepIndex = 1
-            break
-          case ('chief'):
-            stepIndex = 2
-            break
-          case ('head'):
-            stepIndex = 3
-            break
-        }
-      }
-      return stepIndex
-    },
-
-    isUpVoteElectionBannerVisible () { return this.upvoteElection?.isActive },
-
     isWelcomeBannerVisible () { return true },
```

Comment:
@evgenibir I know this is old code, however It should be pulled from some remote config / env variable